txclib/project.py

```diff
@@ -312,7 +314,9 @@ def get_resource_lang_mapping(self, resource):
             except configparser.NoOptionError:
                 pass
             except (ValueError, KeyError):
-                raise Exception("Your lang map configuration is not correct.")
+                raise Exception(
```

Comment:
Since this is exception handling is used more than one, I would prefer having a specific exception type with a default exception message.

Reply:
@fathineos I used MalformedConfigFile instead. Might be a bit of an overkill to introduce yet another exception for just two places where it is handled.